### ISSUE TITLE
Support Highlighting in IL views

### ISSUE THREAD

**Issue by erhlee-bird** [Vector35/binaryninja-api#811]
I can highlight disassembly basic blocks, but cannot for at least medium level il blocks.

```
>>> current_function.medium_level_il.basic_blocks[0].highlight
<color: none, alpha 0>
>>> current_function.medium_level_il.basic_blocks[0].highlight = HighlightStandardColor.BlueHighlightColor
>>> current_function.medium_level_il.basic_blocks[0].highlight
<color: none, alpha 0>
```

vs

```
>>> current_function.basic_blocks[0].highlight
<color: none, alpha 0>
>>> current_function.basic_blocks[0].highlight = HighlightStandardColor.BlueHighlightColor
>>> current_function.basic_blocks[0].highlight
<color: blue>
```

Binary Ninja Version: 1.1.923-dev, 7bf8ca42
Platform: Linux

**Comment by erhlee-bird** [comment]
This behavior is consistent with right clicking and setting block colors in the GUI as well.

**Comment by psifertex** [comment]
Correct -- highlighting is not currently supported in these views. Marking as 1.2.

**Comment by psifertex** [comment]
Thanks -- I thought we had a bug tracking this already but couldn't find it.

**Comment by erhlee-bird** [comment]
No worries, thanks for the quick response!

**Comment by plafosse** [comment]
Added in v1.1.1428

**Comment by yuv422** [comment]
This doesn't appear to be working for me in `Version 1.1.1461-dev Personal (Build ID 435d8566)`

Platform MacOSX

```
current_function.medium_level_il.basic_blocks[0].highlight
<color: none, alpha 0>
 current_function.medium_level_il.basic_blocks[0].highlight = HighlightStandardColor.BlueHighlightColor
current_function.medium_level_il.basic_blocks[0].highlight
<color: none, alpha 0>
```

**Comment by psifertex** [comment]
Yeah, I can confirm it's not working as well. Re-opening.

**Comment by rssor** [comment]
Resolved as of build 1.1.1543-dev. It is worth noting that highlights are only stored/loaded in terms of the native block, so all multiple IL blocks derived from one native block will share a highlight.
